### PR TITLE
Human readable list of notable updates since last release. 

### PR DESCRIPTION
**Pull request by prckent**
Note that my NEXUS description is lacking.

Please update with whatever I missed that is considered important.

Note that it should be possible to edit this update directly on GitHub.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,53 @@
...
+* All particle move VMC and DMC algorithms enabled, tests added
+* Reptation Monte Carlo (RMC) enabled, tests added
+* Significantly improved AFQMC implementation
+* Added NiO based VMC and DMC performance tests and description in manual. Files accessed via QMC_DATA
+* Added DMC tests with locality and t-moves aapproximations
+* Added AFQMC tests
+* Added test of real space QMC restart capabilities
+* Added tests for several estimators
+* Added unit test for DMC walker propagation, effective core potentials, OhmmsPETE
+* To avoid filesystem limitations, QMC_SYMLINK_TEST_FILES can be set to
```
Comment (markdewing) [inline CHANGELOG.md]: This sentence seems unfinished.

Comment (prckent) [inline CHANGELOG.md]: Fixed in update

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,53 @@
...
+* Reptation Monte Carlo (RMC) enabled, tests added.
+* Significantly improved AFQMC implementation.
+* Added NiO based VMC and DMC performance tests and description in manual. Wavefunction files accessed via QMC_DATA.
+* Added DMC tests with locality and t-moves aapproximations.
+* Added AFQMC tests.
+* Added test of real space QMC restart capabilities.
+* Added tests for several estimators.
+* Added unit test for DMC walker propagation, effective core potentials, OhmmsPETE.
+* To avoid filesystem limitations, QMC_SYMLINK_TEST_FILES can be set to symlink (1) or copy test files (0).
+* Status of t-moves now printed.
```
Comment (ye-luo) [inline CHANGELOG.md]: I think this was reverted by me. I'm still waiting for Ray to resubmit a better way printing.

**Comment by jtkrogel** [comment]
Hmm, last update clobbered Paul's "Improve description" update.  Perhaps simultaneous edit + commit directly in github is dangerous...

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,55 @@
...
+since the last release.
+
+### QMCPACK updates
+
+* Improved population control during DMC equilibration. Reduces variance on larger runs.
+* All particle move VMC and DMC algorithms enabled, tests added
+* Reptation Monte Carlo (RMC) enabled, tests added
+* Significantly improved AFQMC implementation
+* Added NiO based VMC and DMC performance tests and description in manual. Files accessed via QMC_DATA
+* Added DMC tests with locality and t-moves aapproximations
```
Comment (ye-luo) [inline CHANGELOG.md]: Typo approximations.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,55 @@
...
+* Added AFQMC tests
+* Added test of real space QMC restart capabilities
+* Added tests for several estimators
+* Added unit test for DMC walker propagation, effective core potentials, OhmmsPETE
+* To avoid filesystem limitations, QMC_SYMLINK_TEST_FILES can be set to symlink (1) or copy test files (0)
+* Status of t-moves now printed
```
Comment (ye-luo) [inline CHANGELOG.md]: This was reverted. Waiting for a new PR from Ray.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,55 @@
...
+* Fixed mixed precision Ceperley force evaluation
+* Many updated tests to improve statistical reliability. Removed flux estimator from short tests because they were not reliable enough
+* Tests that rely on non-standard python modules that are not available are skipped
+* Error trap jastrow factors with cutoff radii larger than Wigner Seitz radius
+* Bugfix: 2-body jastrow on GPU with no alternate spin particles
```
Comment (ye-luo) [inline CHANGELOG.md]: This more error trapping. Prevent users adding correlation terms on non-existing electron pairs on both CPU and GPU. The bug fix on GPU is only a side effect.

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,55 @@
...
+* Support for measuring test coverage and performing coverage runs with cmake and ctest
+* Support for GCC7 and IBM XL compiler
```
Comment (ye-luo) [inline CHANGELOG.md]: Better mention not for BG/Q

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,55 @@
...
+* SummitDev IBM Minsky build recipe (Power8 + NVIDIA Pascal P100 GPUs)
```
Comment (ye-luo) [inline CHANGELOG.md]: Mention CUDA_ARCH for switching to any architecture beyond sm_35.

File: CHANGELOG.md (hunk 2 above)
Comment (prckent) [inline CHANGELOG.md]: Removed

File: CHANGELOG.md (hunk 4 above)
Comment (prckent) [inline CHANGELOG.md]: Same comment as above, removed

File: CHANGELOG.md (hunk 6 above)
Comment (prckent) [inline CHANGELOG.md]: Fixed

**Comment by prckent** [comment]
Updated with results of #272 which should be merged before this PR.

**Pull request merged**